### DISCUSSION TITLE
Run as root

### DISCUSSION THREAD

**Discussion by dazbobaby** [fastfetch-cli/fastfetch#892] [Q&A]
I've installed homebrew and fastfetch as non-root user and it works fine. But how do I run and configure fastfetch as root? 
TIA.
Daz.
BTW running deb.

**Comment by CarterLi** [comment]
Run fastfetch as root? `sudo fastfetch`?

**Comment by dazbobaby** [comment]
I added `eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"` to root's .bashrc
Runs fine now.
